releasenotes/notes/fix-peer-service-module-efec5b701bea2752.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    internal: This fix resolves an issue where importing the internal/peer_service module would fail raising an ImportError
```

Comment:
```suggestion
    internal: This fix resolves an issue where importing the ``ddtrace.internal.peer_service`` module would fail raising an ImportError
```